src/components/molecules/Settings/Project/EditableItem/index.tsx

```diff
@@ -24,6 +29,8 @@ export type Props<T extends string = string> = {
   currentItem?: T;
   imageSrc?: string;
   icon?: string;
+  isAvatar?: boolean;
+  avatar?: string;
```

Comment:
`isAvatar` isn't necessary since you are passing `avatar` as well. Where you are using isAvatar, just do `!!avatar ? ... : ...` instead!

Reply:
Just to note (and not something you need to do or should do in this PR): 

I think this component really needs a refactor. For now adding the avatar prop is fine. But ideally this component might be better with a React.componentType prop. That way we could pass an `<Icon />` or `<Avatar />` or something else and get rid of several props. 
Of course, there might be better ideas out there since this is just off the top of my head haha